app/schemas/models/user.coffee

```diff
@@ -108,6 +108,7 @@ _.extend UserSchema.properties,
         type: c.shortString() # E.g 'subscribe modal parent'
         email: c.shortString()
         sent: c.date() # Set when sent
+  unsubscribedFromEmails: { type: 'boolean' }
```

Comment:
As discussed offline, we should rename this to `unsubscribedFromMarketingEmails` to avoid confusion.

Reply:
Done